src/main-process/atom-application.coffee

```diff
@@ -812,3 +814,10 @@ class AtomApplication
       args.push("--resource-path=#{@resourcePath}")
     app.relaunch({args})
     app.quit()
+
+  clone = (obj) ->
```

Comment:
Here a recursive clone is needed, so neither of these functions will work, but maybe [`_.cloneDeep`](https://lodash.com/docs/4.17.4#cloneDeep) could be used.